test/e2e/scripts/deploy-platform.sh

```
@@ -0,0 +1,136 @@
+            echo "❌ ERROR: Keycloak test realm import failed (see docs/samples/install/keycloak/test-realms/)"
+            exit 1
+        fi
+
+        echo "Mounting ingress CA certificate into Authorino for OIDC JWKS discovery..."
+        local ingress_cert_name
+        ingress_cert_name=$(oc get ingresscontroller default -n openshift-ingress-operator \
+            -o jsonpath='{.spec.defaultCertificate.name}' 2>/dev/null)
+        if [[ -n "$ingress_cert_name" ]]; then
+            local ca_tmp
+            ca_tmp=$(mktemp)
+            trap "rm -f '$ca_tmp'; trap - RETURN" RETURN
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)